mount/mount.go

```diff
@@ -69,6 +70,16 @@ func UnmountMounts(mounts []Mount, target string, flags int) error {
 	return nil
 }
 
+func CanonicalizePath(path string) (string, error) {
+	// Abs also does Clean, so we do not need to call it separately
+	path, err := filepath.Abs(path)
+	if err != nil {
+		return "", err
+	}
+
+	return filepath.EvalSymlinks(path)
```

Comment:
If path does not exist, you won't find it in mount table and it possibly a logic error that code is even trying to do that.

Additionally, attempt to handle nonexistent path makes things much more complex because _part of path_ could exist and involve symlinks. Should we resolve those or not?

Overall, I'd keep this function as is for now. Do you want me to add a doc string to it?

Reply:
Done